sympy/utilities/lambdify.py

```diff
@@ -9,6 +9,7 @@
 import keyword
 import textwrap
 import linecache
+from  .exceptions import SymPyDeprecationWarning
```

Comment:
I don't know if we have an explicit policy on this, but we generally use absolute imports in SymPy. And regardless, this should go in the block below with the SymPy imports. Having it with the standard library imports is confusing.

Reply:
I didn't notice that this was already absolute, but changed in response to https://github.com/sympy/sympy/pull/19792#discussion_r468438379. I'm not really sure that `from .exception import ...` would be any different from `from sympy.utilities.exception ...` with respect to circular imports. Python's import semantics are complicated, but I believe both will do the same thing. As I noted, absolute imports are used almost everywhere in SymPy, except in `__init__.py` files. Actually I checked and we aren't really consistent. It might be best to open a new issue to discuss this.